recommenders/models/rlrmc/RLRMCdataset.py

```diff
@@ -68,8 +68,8 @@ def _data_processing(self, train, validation=None, test=None, mean_center=True):
         """
         # Data processing and reindexing code is adopted from https://github.com/Microsoft/Recommenders/blob/main/recommenders/models/ncf/dataset.py
         # If validation dataset is None
-        df = train if validation is None else train.append(validation)
-        df = df if test is None else df.append(test)
+        df = train if validation is None else pd.concat([train, validation])
+        df = df if test is None else pd.concat([df, test])
```

Comment:
You can avoid having two concats with this one liner:

`df = pd.concat(filter(None, [train, validation, test]))`

Reply:
hey @daviddavo, can you please create a PR to staging with these changes? I gave you written permissions.
```
git checkout staging
git branch -b new_branch
.... code changes ...
git commit -asm "you message don't forget the -s to sign the commits"
git push origin new_branch
```
And then PR.